utils/files.c

```diff
@@ -5,6 +5,12 @@
  * Refer to the LICENSE file for licensing information
  */
 
+#ifdef _WIN32
```

Comment:
could you do a check similar to the one in the threading utils there?

Reply:
Sure